src/test/java/UsersBookTest.java

```diff
@@ -0,0 +1,72 @@
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class UsersBookTest {
+
+    UsersBook usersBook;
+    User user1;
+    User user2;
+    User user3;
+
+
+    @BeforeEach
+    void setUp() {
+        usersBook = new UsersBook();
+        user1 = new User("name1", "surname1", 20);
```

Comment:
why you need to create new user before each test ? Would it be enough create them once ?